integration_tests/buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -0,0 +1,139 @@
+object Versions {
```

Comment:
do we need all these dependencies?

Reply:
was going to ask the same thing

Reply:
And if we do, we should check if they will always match with what is done in flank test_runner and if it does then we should have a more global build config

Reply:
infact if i am correct @jan-gogo is working on a mono gradle repo and i am sure this would solve it :)